packages/host/app/components/operator-mode/detail-panel.gts

```diff
@@ -397,8 +397,14 @@ export default class DetailPanel extends Component<Signature> {
 
 const resolveElementName = (el: Element) => {
   let localName: string | undefined = el.localName;
+  let type: 'card' | 'field' | undefined;
   if (isCardOrFieldElement(el)) {
     localName = el.cardOrField.displayName;
+    type = isFieldDef(el.cardOrField) ? 'field' : 'card';
   }
-  return localName ?? '[No Name Found]';
+  let exportedName = el.exportedAs;
+  let resolvedName = `${localName ?? '??'} 
+  (type: ${type ?? '??'}) 
+  (exported name: ${exportedName ?? '??'})`;
+  return resolvedName;
```

Comment:
PS. I intentionally changed this for ease of review. Its is one of those things I think may be easier to consider when we handle the 

https://linear.app/cardstack/issue/CS-5993/handle-situation-where-each-definition-has-the-same-name